src/components/recipeList.test.tsx

```diff
@@ -1,10 +1,11 @@
 import React from "react";
 import { render, screen } from "@testing-library/react";
+import { MemoryRouter } from "react-router-dom";
 import RecipeList from "./recipeList";
 
 describe("when rendered", () => {
```

Comment:
It would be nice to have a test which confirms that recipe list includes a link to the correct recipe, especially once it fetches data from the back-end API.

Reply:
I'll add that to the next ticket